test/JDBC/input/BABEL-3486-vu-prepare.sql

```diff
@@ -0,0 +1,161 @@
+-- View Conversions with TRY_CONVERT
+CREATE VIEW BABEL_3486_vu_prepare_v1 as (SELECT TRY_CONVERT(varchar(30), CAST('13:01:59' AS time), 8));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v2 as (SELECT TRY_CONVERT(date, '2017-08-25'));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v3 as (SELECT TRY_CONVERT(time, '13:01:59'));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v4 as (SELECT TRY_CONVERT(datetime, '2017-08-25 13:01:59'));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v5 as (SELECT TRY_CONVERT(float, 11234561231231.234));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v6 as (SELECT TRY_CONVERT(MONEY, 55.35));
+GO
+
+CREATE VIEW BABEL_3486_vu_prepare_v7 as (SELECT TRY_CONVERT(float, 1));
```

Comment:
Have we covered all the datatypes? Can think of a few more off the top of my head - numeric, tinyint, sql_variant. I think we should look up supported datatypes and cover the following cases on each of them:
1. Valid operation
2. Invalid operation
3. Edge case